tools/benchmark/src/mocha/runner.ts

```diff
@@ -110,11 +110,16 @@ export function supportParentProcess<
 			const childProcess = await import("node:child_process");
 			const result = childProcess.spawnSync(command, childArgs, { encoding: "utf8" });
 
+			if (result.stdout) {
+				console.log(result.stdout);
+			}
```

Comment:
In light of the changes above, we can't do this. Further down we depend on `stdout` from child processes having nothing but a JSON payload; that's basically the interface/contract between parent and child processes. Adding plaintext output to that will probably break something else. Try to see if there's a way of communicating the warning within the existing payload that child processes write to their output. If not, let's discuss how/if we can keep the warning.